spec/features/user_signs_in_spec.rb

```diff
@@ -179,4 +179,24 @@
     end
 
   end
+
+  scenario 'with an email address and password' do
+    with_forgery_protection do
+      create_application
+      user = create_user 'user'
+      create_email_address_for user, 'user@example.com'
+      visit_authorize_uri
+      expect(page).to have_content("Sign in to your one OpenStax account!")
+
+      fill_in 'Username', with: 'user'
+      fill_in 'Password', with: 'pass'
+      click_button 'Sign in'
+      expect(page).to have_content('Incorrect username or password')
```

Comment:
Since we allow username or email address, can we make this error message more generic?  Per JH, change to "Incorrect username, email, or password."